pos_partner_deselection/README.rst

```diff
@@ -2,11 +2,11 @@
    :target: https://www.gnu.org/licenses/lgpl
    :alt: License: LGPL-3
 
-==================
- Deselect Partner
-==================
+====================
+Partner  deselection
+====================
 
-POS Partner deselection after a defined time interval
+The module allows to deselect partner in opened POSes by the user after a defined time interval.
```

Comment:
why do we need "by user" here? It's always a user who opens POS. Also, not sure that POS can be plural here

Reply:
got it